EMS/core-bundle/src/DataTable/Type/ChannelDataTableType.php

```diff
@@ -21,16 +22,40 @@ public function __construct(ChannelService $entityService)
 
     public function build(EntityTable $table): void
     {
-        $table->addColumn('table.index.column.loop_count', 'orderKey');
-        $table->addColumn('channel.index.column.label', 'label');
-        $column = $table->addColumn('channel.index.column.name', 'name');
-        $column->setPathCallback(fn (Channel $channel, string $baseUrl = '') => $baseUrl.$channel->getEntryPath(), '_blank');
-        $table->addColumn('channel.index.column.alias', 'alias');
-        $table->addColumnDefinition(new BoolTableColumn('channel.index.column.public', 'public'));
-        $table->addItemGetAction('ems_core_channel_edit', 'channel.actions.edit', 'pencil');
-        $table->addItemPostAction('ems_core_channel_delete', 'channel.actions.delete', 'trash', 'channel.actions.delete_confirm');
-        $table->addTableAction(TableAbstract::DELETE_ACTION, 'fa fa-trash', 'channel.actions.delete_selected', 'channel.actions.delete_selected_confirm');
         $table->setDefaultOrder('orderKey');
+
+        $table->addColumn(t('table.loop_count', [], 'emsco-core'), 'orderKey');
+        $table->addColumn(t('field.label', [], 'emsco-core'), 'label');
+
+        $column = $table->addColumn(t('field.name', [], 'emsco-core'), 'name');
+        $column->setPathCallback(fn (Channel $channel, string $baseUrl = '') => $baseUrl.$channel->getEntryPath(), '_blank');
+
+        $table->addColumn(t('field.alias', [], 'emsco-core'), 'alias');
+        $table->addColumnDefinition(new BoolTableColumn(t('admin.channel.field.public', [], 'emsco-core'), 'public'));
+
+        $table->addItemGetAction(
+            route: 'ems_core_channel_edit',
+            labelKey: t('action.edit', [], 'emsco-core'),
+            icon: 'pencil'
+        );
+        $table->addItemPostAction(
+            route: 'ems_core_channel_delete',
+            labelKey: t('action.delete', [], 'emsco-core'),
```

Comment:
I think there is a constant for 'emsco-core'

Reply:
Using the t() function with a constant makes that the crawler will not detect the key. To make our translations automatically build with ```php  build/translations en EMSCoreBundle --write --format=yml -d emsco-core``` we can not use constants as translation domain.